### PR TITLE
[1/x] split project page into project info component

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by jmilldotdev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1714** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1714" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1714?utm_source=stack-comment).

File: src/components/v2/V2Project/V2ProjectHeaderActions.tsx [hunk 1]
```diff
@@ -61,6 +75,15 @@ export default function V2ProjectHeaderActions() {
           />
         </Tooltip>
         {showReconfigureButton && <V2ReconfigureFundingModalTrigger />}
+        {settingsPageEnabled && (
+          <Tooltip title={t`Open settings page`} placement="bottom">
+            <Button
+              icon={<InfoCircleOutlined />}
+              type="text"
+              onClick={handleSettingsPageButtonClick}
```
Comment (tomquirk) [inline src/components/v2/V2Project/V2ProjectHeaderActions.tsx:83]: **suggestion** for accessibility, lets make this a real link. Not exactly sure if Next Link will work, but `<Button href="/v2/p/${projectId}/settings">` does.

File: src/pages/v2/p/[projectId]/components/V2Dashboard.tsx [hunk 2]
```diff
@@ -223,7 +223,7 @@ export default function V2Dashboard({
   return (
     <V2ProjectContext.Provider value={project}>
       <div style={layouts.maxWidth}>
-        <V2Project />
+        <V2Project selectedPage="info" />
```
Comment (tomquirk) [inline src/pages/v2/p/[projectId]/components/V2Dashboard.tsx:226]: **suggestion** can we use the URL as a single source of truth? In other words, can we use the URL alone to determine the render state?

File: src/components/v2/V2Project/index.tsx [hunk 3]
```diff
@@ -162,97 +73,17 @@ export default function V2Project({
...
-        storeCidTx={editV2ProjectDetailsTx}
-      />
-      {showAddHandle && (
-        <V2ReconfigureProjectHandleDrawer
-          visible={handleModalVisible}
-          onFinish={() => setHandleModalVisible(false)}
+      <Space direction="horizontal">
+        <Button onClick={() => setActivePage('info')}>Show Info</Button>
+        <Button onClick={() => setActivePage('settings')}>Show Settings</Button>
+      </Space>
```
Comment (tomquirk) [inline src/components/v2/V2Project/index.tsx:79]: **question** is this the final design?

**Review by tomquirk — CHANGES_REQUESTED** [review]

**Comment by tomquirk** [comment]
@jmilldotdev looking good!

I'm not sure if there's changes to routing later on in the stack. If there are, I think we should consolidate them to this PR. Let's get this PR to:
- reorganize the files appropriately (V2ProjectInfo etc)
- Include the final button UI (for getting to and from the settings page)
- Handle routing and page-switching logic.

That way we set a solid foundation for the PRs up the stack 🙌

**Comment by tomquirk** [comment]
Closing in favor of https://github.com/jbx-protocol/juice-interface/pull/1875

**Pull request closed without merging**